server/users/models.py

```diff
@@ -0,0 +1,28 @@
+import uuid
+from typing import override
+
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
```

Comment:
we can write tests for the user model

Reply:
Pushed some tests
Will push more for the remaining fields to be added